pkgs/development/python-modules/simplekv/default.nix

```diff
@@ -40,11 +45,28 @@ buildPythonPackage rec {
     "test_concurrent_mkdir"
   ];
```

Comment:
```diff
++ optional-dependencies.git
```

Could you try adding more optional dependencies to the `nativeCheckInputs` so more tests are run?